services/src/test/java/org/exoplatform/addons/gamification/search/RuleSearchConnectorTest.java

```diff
@@ -99,7 +100,7 @@ public void testSearch() {
     filter.setSpaceIds(listIdSpaceEmpty);
     assertThrows(IllegalArgumentException.class, () -> ruleSearchConnector.search(filter, 0, 10));
     filter.setSpaceIds(listIdSpace);
-
+    filter.setFilterType(FilterType.STARTED);
```

Comment:
It's better to not change an existing Test only when mandatory and add new ones to increase the coverage with new Test Cases.